infinispan-client-quickstart/README.md

```diff
@@ -6,15 +6,14 @@ This example showcases how to use the Infinispan client with Quarkus.
 
 You have two options:
 
-**Option 1:** Running with Docker `docker run -it -p 11222:11222 jboss/infinispan-server:latest`
+**Option 1:** Running with Docker `docker run -it -p 11222:11222 -e USER="Titus Bramble" -e PASS="Shambles" infinispan/server:latest`
```

Comment:
You have a space in the user name here and not in the Docker file. I don't think that's what you want?

Reply:
I changed for the space one